src/Email.php

```diff
@@ -61,4 +109,43 @@ public static function signQuery($query) {
     $key = drupal_get_hash_salt() . __FILE__;
     return drupal_hmac_base64(serialize($query), $key);
   }
+
+  /**
+   * Adds a widget containing the options of the `Email` channel.
```

Comment:
Again: This doesn’t "add a widget" it allows the channel to display additional configuration form elements *inside* the field widget.